website/docs/gen-ai/inference/Llama2.md

```diff
@@ -0,0 +1,310 @@
+---
+title: Llama-2 on EKS
+sidebar_position: 1
+---
+import CollapsibleContent from '../../../src/components/CollapsibleContent';
+
+
+:::danger
+
+Note: Use of this Llama-2 model is governed by the Meta license.
+In order to download the model weights and tokenizer, please visit the [website](https://ai.meta.com/) and accept our License before requesting access here.
+
+:::
+
+:::info
+
+We are actively enhancing this blueprint to incorporate improvements in observability, logging, and scalability aspects.
+
+:::
+
+
+# Llama-2-Chat on EKS: Deploying Llama-2-13b Chat Model with Ray Serve and Gradio
+Welcome to the comprehensive guide on deploying the [Meta Llama-2-13b chat](https://ai.meta.com/llama/#inside-the-model) model on Amazon Elastic Kubernetes Service (EKS) using [Ray Serve](https://docs.ray.io/en/latest/serve/index.html).
+In this tutorial, you will not only learn how to harness the power of Llama-2, but also gain insights into the intricacies of deploying large language models (LLMs) efficiently, particularly on [AWS Neuron](https://aws.amazon.com/machine-learning/neuron/) instances, such as `inf2.24xlarge` and `inf2.48xlarge`, which are optimized for deploying and scaling large language models.
```

Comment:
AWS Neuron is the SDK. We prefer to call-out trn1/inf2 instances (powered by AWS Trainium and Inferentia) rather than brand the instances as 'Neuron'.